/**:
  ros__parameters:
    port: "/dev/ttyACM0"
    baud_rate: 57600
    encoder_rate: 10
    timeout: 1.0
    wheel_separation: 0.3
    wheel_radius: 0.05
    encoder_resolution: 360
